src/components/MainSidebar/DimensionItem/DimensionItem.js

```diff
@@ -48,11 +47,7 @@ export const DimensionItem = ({
     } = useSortable({
         id: draggableId || id,
         disabled: disabled || selected,
-        data: {
-            name,
-            dimensionType,
-            valueType,
-        },
+        data: dimensionMetadata,
```

Comment:
Note that I also changed this. It's an attempt to standardise which properties are present in the dimension metadata.